data_steward/gcloud/bq/__init__.py

```diff
@@ -184,3 +183,86 @@ def define_dataset(self, dataset_id: str, description: str,
         dataset.location = "US"
 
         return dataset
+
+    def copy_datasets(self, input_dataset: str, output_dataset: str):
```

Comment:
nit: Since only one dataset can be copied at a time
```suggestion
    def copy_dataset(self, input_dataset: str, output_dataset: str):
```